Solutions/OpenCTI/Playbooks/OpenCTIPlaybooks/OpenCTI-GetIndicatorsStream/azuredeploy.json

```diff
@@ -1,31 +1,39 @@
-        "3. OpenCTI-ImportToSentinel must be installed and keep playbook name and batchname handy"],
-        "postDeployment":  ["1. Once deployment is complete, you will need to authorize each connection.",
-        "2. Navigate to playbook --> API Connections --> Select connections one by one --> Edit API Connection --> (if required) Enter API key or credentials --> Save"],
-        "prerequisitesDeployTemplateFile":  "../../CustomConnector/OpenCTICustomConnector/azuredeploy.json",
-        "lastUpdateTime":  "2022-07-19T00:00:00.000Z",
-        "entities":  ["account","ip","filehash","url","host"],
-        "tags":  ["Stream","Ingestion","ThreatIntelligence", "OpenCTI"
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "metadata": {
+        "title": "Read Stream- OpenCTI Indicators",
+        "description": "This playbook fetches indicators from OpenCTI and send to Sentinel. Supported types are Domain, File, IPv4, IPv6, Account, Url.  This runs for every 10 minutes",
+        "prerequisites": [
+            "1. OpenCTI Custom Connector needs to be deployed prior to the deployment of this playbook under the same subscription.",
+            "2. API key. To get API Key, login into your OpenCTI instance dashboard and navigate to User profile page --> API Access.",
+            "3. OpenCTI-ImportToSentinel must be installed and keep playbook name and batchname handy"
         ],
-        "support":  {
-            "tier":  "microsoft",
-            "armtemplate":  "Generated from https://github.com/Azure/Azure-Sentinel/tree/master/Tools/Playbook-ARM-Template-Generator"
+        "postDeployment": [
+            "1. Once deployment is complete, you will need to authorize each connection.",
+            "2. Navigate to playbook --> API Connections --> Select connections one by one --> Edit API Connection --> (if required) Enter API key or credentials --> Save"
+        ],
+        "prerequisitesDeployTemplateFile": "../../CustomConnector/OpenCTICustomConnector/azuredeploy.json",
+        "lastUpdateTime": "2022-07-19T00:00:00.000Z",
+        "entities": [ "account", "ip", "filehash", "url", "host" ],
+        "tags": [
+            "Stream",
+            "Ingestion",
+            "ThreatIntelligence",
+            "OpenCTI"
+        ],
+        "support": {
+            "tier": "microsoft",
+            "armtemplate": "Generated from https://github.com/Azure/Azure-Sentinel/tree/master/Tools/Playbook-ARM-Template-Generator"
         },
-        "author":  {
-            "name":  "Microsoft"
+        "author": {
+            "name": "Microsoft"
         }
     },
-    "parameters":  {
-        "PlaybookName":  {
-            "defaultValue":  "OpenCTI-IndicatorsStream",
-            "type":  "string"
+    "parameters": {
+        "PlaybookName": {
+            "defaultValue": "OpenCTI-IndicatorsStream",
```

Comment:
Default Playbook name should match to folder name for consistency. Here folder name is "OpenCTI-GetIndicatorsStream" but default playbook name is "OpenCTI-IndicatorsStream".